public/js/main.js

```diff
@@ -411,4 +427,10 @@ $(function () {
       dataType: 'json'
     })
   }
+
+  function getCookieValue(a) {
```

Comment:
Please move this above the first call to this method and use this.`var getCookieValue = function(a){};